### ISSUE TITLE
Trouble getting toolkit to work, even examples

### ISSUE THREAD

**Issue by NickDevoctomy** [Microsoft/MixedRealityToolkit-Unity#1682]
## Overview

I'm having great difficulty getting any of this to run on my Acer WMR Headset.  I'm not sure what's causing the issue, but anything with the Toolkit included for me, won't run on my headset.

I have Steam VR installed, and am not entirely sure how much that intervenes, but for example, I *can* do the following,

1.  Create a Unity project using standard assets
2.  Run the project on my headset, it works fine

-

3.  Add the MixedRealityToolkit and run through the getting started guide to configure my project, created previously.
4.  Remove my FPS controller and replace it for the MixedRealityCamera
5.  Run...
6.  Here it just won't show on my headset but it says "The Fade Manager singleton has not been initialized".  The same goes for any scenes from the Toolkit examples.  I'm not sure if this is all that's stopping it or not.

## Steps to reproduce

So I can produce this pretty easily,

1.  Create new Unity 3D Project
2.  Add the Toolkit and Examples Unity packages.
3.  Run Mixed Reality Toolkit > Configure > Apply Mixed Reality Project Settings, 
4.  Select everything except Enable Sharing Services. (It has issues applying here, and sometimes needs Unity to be restarted before the Hololens SDK is available in the player settings.
5.  Open sample scene.
6.  Press Run
7.  Something starts to happen then Unity pauses with error, "The Fade Manager singleton has not been initialized".
8.  If Steam VR is running I get left in limbo land, and if not I just stay in the cliff house.

Not entirely sure how to continue, but I could do with using the Toolkit for motion controller and boundary support.

## Unity Editor Version

2017.1.0.p5

## Mixed Reality Toolkit Release Version

2017.2.1.1

**Comment by StephenHodgson** [comment]
I'd use a more recent Unity Editor version.

**Comment by NickDevoctomy** [comment]
Okay, I'll give that a go thanks.

**Comment by NickDevoctomy** [comment]
Great thanks, that seems to work, a few issues with some of the samples but then I haven't read the README docs.  BTW, does the Acer headset do spatial mapping?  Like maybe I'm misunderstanding this, but can it work out where my desk etc is and allow me to put a mesh on it?

That sample didn't seem to work, so wasn't sure if that's because my headset doesn't support it or not.

**Comment by StephenHodgson** [comment]
We're working on getting the samples updated.  It might be part of the next release. Keep your eyes out for it.

**Comment by Weasy666** [comment]
[Spatial Mapping](https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping) is only available on the HoloLens, the Mixed Reality Headset don't have the sensors for that.

**Comment by NickDevoctomy** [comment]
@StephenHodgson excellent, do you have any idea what the sensors are on the Acer headset? Are they infrared cameras? If so, is there a way the the toolkit can be used to get raw images from them? My bad if this question isn't suited to here.

**Comment by Weasy666** [comment]
Nope, just plain old B&W cameras. If [Acer](https://us-store.acer.com/virtual-mixed-reality/acer-windows-mixed-reality-headset-and-motion-controllers-ah101-d8ey) is not lying ^^

**Comment by NickDevoctomy** [comment]
@Weasy666 damn! :( I thought i might have been wishful thinking.

**Comment by Weasy666** [comment]
I don't know if it is possible to get the video feed of the cameras, but if you can get them you could try to implement something like [visual odometry](https://vision.in.tum.de/research/vslam/dso?redirect=1) and use the points of the point cloud inside unity ^^ but this my be to much of an efford

**Comment by NickDevoctomy** [comment]
@Weasy666 cool idea, looks like I've got some researching to do :)
